packages/react/.storybook/preview.tsx

```diff
@@ -64,7 +64,6 @@ addParameters({
     sort: 'requiredFirst',
   },
   docs: {
-    inlineStories: false,
```

Comment:
why are we doing this?

Reply:
This was added to overcome the stories leaking outside the iframe in components with `fixed` styling.

<img width="1823" alt="Screenshot 2023-02-22 at 12 41 38" src="https://user-images.githubusercontent.com/25959096/220548785-bdb5806d-411a-4c8d-92dc-09a0d7c8f73f.png">

However, this causes the default maxheight of the story window to be removed.

<img width="1048" alt="Screenshot 2023-02-22 at 12 40 33" src="https://user-images.githubusercontent.com/25959096/220548855-789330cd-f1ce-4999-ace2-cceba6ba3f43.png">

Hence, decided to revert the config globally and add in stories selectively.